cloudinstall/charms/ceph.py

```diff
@@ -29,7 +29,8 @@ class CharmCeph(CharmBase):
     display_name = 'Ceph'
     menuable = True
     display_priority = DisplayPriorities.Storage
-    related = ['glance', 'nova-compute']
+    related = {'glance': ('glance:ceph', 'ceph:client'),
+               'nova-compute': ('nova-compute:ceph', 'ceph:client')}
```

Comment:
any particular reason why this is a dictionary instead of just a list of tuples?
The key is always repeated in one of the relation names.